angular/src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,29 @@
+<nav>
+    <div class="nav-wrapper" >
+        <a href="index" class="brand-logo"><img id="logo" src="./assets/images/bttcircle.png" alt="logo"></a>
+        <a href="#" data-target="mobile-demo" class="sidenav-trigger"><i class="material-icons">menu</i></a>
+        <ul class="right hide-on-med-and-down" id="navbar">
+            <li><a href="index">Welcome !</a></li>
```

Comment:
nit: there shouldn't be a space here, it should be "Welcome!"

(it seems like putting a space before punctuation marks is a french thing lol, but it certainly looks weird in english)

Reply:
Oh yes, in french we use a space thats why, I'll correct that